.github/workflows/airbyte-ci-release.yml

```diff
@@ -6,6 +6,15 @@ concurrency:
 
 on:
   push:
+    branches:
+      - master
+    paths:
+      - "airbyte-ci/connectors/pipelines/**"
+  pull_request:
+    types:
+      - opened
+      - reopened
+      - synchronize
     paths:
       - "airbyte-ci/connectors/pipelines/**"
```

Comment:
If I'm not mistaken the release will be triggered on `airbyte-ci` file changes only.
So if you make a change to a non airbyte-ci related file on a later commit the `run-dagger-pipeline/action.yml` will never resolve a binary for this commit.



I think that a dev install with pipx when the PR (not the commit) has `airbyte-ci` modified files is easier to implement.

Another option would be to always trigger this workflow (no `paths` filter) and use `tj-actions/changed-files` to only trigger a release if the branch has airbyte-ci modified files.

I'm not sure if `tj-actions/changed-files` returns the changed-files compared to the previous commit or compared to the target branch

Reply:
> If I'm not mistaken the release will be triggered on airbyte-ci file changes only.
So interestingly enough this was the case previously when using `push.paths`. However its not the case for `pull_request.paths` + `pull_request.types=synchronize`

> I think that a dev install with pipx when the PR (not the commit) has airbyte-ci modified files is easier to implement.
I think your right. I went down this path as it would be useful to test pipeline changes using the final pyinstaller executable. As this is what the end users actually run.

But looking at this now, the steps we take to poll for new dev binaries that match your commit is brittle and more complex than we should tolerate.

TLDR: Ill update to use the pipx dev install instead